schema/project.js

```diff
@@ -55,8 +55,7 @@ class Project extends BaseModel {
 
   static getOwn({ establishmentId, id, licenceHolderId }) {
     return this.query()
-      .where({ establishmentId })
-      .where({ licenceHolderId })
+      .where({ establishmentId, licenceHolderId })
```

Comment:
Is this just a style change or does it affect the query results?

Reply:
Just a style change, .where .where seems a little ambiguous as it's not immediately clear if its .andWhere or .orWhere

Reply:
Just a style change.